eng/build/Engineering.props

```diff
@@ -1,5 +1,12 @@
 <Project>
 
+  <!-- artifacts -->
+  <PropertyGroup>
+    <ArtifactsPath>$(RepoRoot)out</ArtifactsPath>
```

Comment:
Should it be `$(RepoRoot)/out` here?

Reply:
`RepoRoot` will have the trailing slash as part of it already (this is a common pattern in msbuild, not sure why they do that, but they do)